amundsen_application/static/css/_layouts.scss

```diff
@@ -102,6 +102,10 @@ $resource-header-height: 84px;
 
       .editable-text {
         font-size: 16px;
+        
+        .markdown-wrapper{
+          font-size: 16px;
+        }
```

Comment:
Out of curiosity, is this needed?
Usually the statement above will do, but it could be that there are some style overrides.

Reply:
Yes it has been overridden, I will change font there and see if that is the correct way to do it.

Reply:
I'd say this option was a better fix. `EditableText` is used in our column descriptions as well, which I believe are to remain at `14px`. We can keep the default at `14px` for that component and instead only apply `16px` where needed -- which this version looks like it accomplishes.

Reply:
@dikshathakur3119 FYI see comment above.

Reply:
Thanks for the update.